components/heap/multi_heap_internal.h

```diff
@@ -28,6 +31,7 @@ void multi_heap_get_info_impl(multi_heap_handle_t heap, multi_heap_info_t *info)
 size_t multi_heap_free_size_impl(multi_heap_handle_t heap);
 size_t multi_heap_minimum_free_size_impl(multi_heap_handle_t heap);
 size_t multi_heap_get_allocated_size_impl(multi_heap_handle_t heap, void *p);
+void* multi_heap_get_block_address_impl(multi_heap_block_handle_t block);
```

Comment:
This is minor, but in the rest of the codebase we usually write `void *` not `void* `. (I see we're not 100% consistent on this ourselves, though!) Don't mind in the source files, but do you mind changing the headers?

Reply:
I've done more C++ than C for a while, hence the preference for void*, but I'm happy to change this.

Reply:
Addressed in 5683a7c.